### PR TITLE
Fixed bugs associated with the camera

### PR DESCRIPTION
**Pull request by JiepengTan**
fixed: https://github.com/goplus/spx/issues/418

test project: ./test/Bananas

camera_bugs

### CODE REVIEW

File: test/Bananas/.config
Comment (nighca) [inline test/Bananas/.config:1]: Is this file expected to be commited?

Comment (JiepengTan) [reply test/Bananas/.config:1]: For SPX2, this config is essential because it involves the integration of external res dir.

Comment (nighca) [reply test/Bananas/.config:1]: If an spx2 project references some resource file outside its folder, a `.config` file is required in the project folder.

Is that correct?

Comment (JiepengTan) [reply test/Bananas/.config:1]: This pertains to compatibility with projects under the "test" directory in spx1, 
Most of them need stuff from outside(../res), so in theory, you can only run them on PC

Comment (JiepengTan) [reply test/Bananas/.config:1]: > If an spx2 project references some resource file outside its folder, a `.config` file with `extasset` is required to be placed in the project folder.
> 
> Is that correct?

this is not work in web platform ( - - !)

**Review by nighca — APPROVED** [review]

**Pull request merged**